services/store/scripts/deploy.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+if [ -z $1 ]; then
+    echo "Usage: $(basename $0) CONTAINER_REGISTRY" >&2
+    exit 1
+fi
+
+root=`cd $(dirname $0); cd ..; pwd`
+
+version=`cat $root/version`
+svc_name=`basename $root`
+
+container_reg=$1
+image_name="$container_reg.azurecr.io/$svc_name:$version"
+
+az acr login -n $container_reg
+docker build -t $image_name $root
+docker push $image_name
+
+
+echo "apiVersion: apps/v1beta1
+kind: Deployment
```

Comment:
>Deployment [](start = 6, length = 10)

Is there a way in which this k8s settings live on their own files?

Reply:
There are replacement strings in the configuration string. We can do that when the configuration is larger.